src/org.mate.marco.gschema.xml

```diff
@@ -202,6 +202,18 @@
         <summary>Icon size</summary>
         <description>Size of the application icons displayed in alt-tab popup window. The screen's scale factor is applied to this value.</description>
     </key>
+    <key name="alt-tab-max-columns" type="i">
+        <range min="1" max="1024"/>
+        <default>5</default>
+        <summary>Maximum number of columns in alt-tab popup window</summary>
+        <description>The popup window may be expanded by empty columns up to the maximum columns width to fit window labels if less columns are displayed and if config option 'alt-tab-max-empty-columns' has a value > 0.</description>
```

Comment:
This description is a bit confusing... maybe change to something like:

> The popup window will be expanded to fit up to these many entries per row.

No need to mention the window titles, as those would be handled separately by the next variable.

Reply:
Ok, I changed it as you suggested, see next commit 06129b4 below.